gradle/dependencies.gradle

```diff
@@ -66,8 +66,8 @@ versions += [
   grgit: "4.1.1",
   httpclient: "4.5.13",
   easymock: "4.3",
-  jackson: "2.13.2",
-  jacksonDatabind: "2.13.2.2",
+  jackson: "2.13.3",
+  jacksonDatabind: "2.13.3",
```

Comment:
Merge conflict for this two dependencies. The other used version was `2.13.2.3` -- I opened to upgrade to `2.13.3` that is used in AK.

Seems we bump the version to `2.13.2.3` in `confluentinc/master` as some point.